### ISSUE TITLE
Allow achievements in online/netstart co-op

### ISSUE THREAD

**Issue by kittenchilly** [UnderscoreKilburn/repentance-issues#1345]
I can't really think of a reason why they are disabled besides a technical one. Modding and debug menu allow achievements now, so netstart should be the same.

**Comment by ZMBanshee** [comment]
Yeah, it's a technical reason. It doesn't use any player's save file, as far as I know it works kind of like dailies do. I'm guessing if there was more dev time put into this feature it would eventually have had that.

**Comment by Rchardon** [comment]
fixed in 1.9.1

**Comment by Rchardon** [comment]
Fixed in Rep+
